bessctl/conf/testing/module_tests/mlfqueue.py

```diff
@@ -0,0 +1,36 @@
+import sugar
+from module import *
+from port import *
+
+def basic_throughput_test():
+    def gen_packet_list(protocol, input_ip, output_ip, num_pkts):
```

Comment:
Can you put a comment here that just says, this generates the input for an output test where you expect the same packet to come out as goes in?